MetaMask/metamask-extension
# UX: Multichain: Send Flow: Recipient -> Content Logic Switch

## **Description**

This PR restructures the parent PR so that there's a duality between the contents shown before the recipient is selected and after.  The stub content ("Content") will be updated in subsequent PRs to show Hex data, asset picker, gas preview, and more.

## **Related issues**

Fixes: #

## **Manual testing steps**

1.  `MULTICHAIN=1 yarn start`
2. Enter the send flow
3. Select an account
4. See the "Your Accounts" and "Contacts" go away, and show stub content

## **Screenshots/Recordings**


https://github.com/MetaMask/metamask-extension/assets/46655/6b3df447-f60f-4dd6-8a86-a5ac1bfd2c4d



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
